### ISSUE TITLE
Install errors - Local machine

### ISSUE THREAD

**Issue by tracychew** [Sydney-Informatics-Hub/dataharvester#3]
My code:

## DataHarvesteR

Install dataharvester

```{r}
install.packages("remotes") # uncomment and run this line if necessary
install.packages("pacman")
remotes::install_github("januarharianto/dataharvester")
```

## Load library

```{r}
pacman::p_load(dataharvester)
initialise_harvester(earthengine=TRUE)
```

Last part of the standard output:

```
Restarting R session...

> knitr::opts_chunk$set(echo = TRUE)
> pacman::p_load(dataharvester)
> initialise_harvester(earthengine=TRUE)

-- Welcome to AgReFed Data-Harvester ------------------------------------------------------------------

<U+2139> Checking if dataharvesteR has been set up appropriately
<U+2139> Check Python/conda install
<e2><9c><94> conda binary: C:/Users/tche7417/AppData/Local/r-miniconda/condabin/conda.bat
<e2><9c><94> conda env: geopy

<U+2139> Checking package versions
Cannot validate required Python packages. Attempting to reinstall all packages to be safe...
Warning in system2(conda, shQuote(args), stdout = TRUE) :
  running command '"C:/Users/tche7417/AppData/Local/r-miniconda/condabin/conda.bat" "list" "--prefix" "C:/Users/tche7417/AppData/Local/r-miniconda/envs" "--no-pip" "--json"' had status 1
Something went wrong here.
<U+2139> Set up Earth Engine API access
Error in py_module_import(module, convert = convert) : 
  ModuleNotFoundError: No module named 'dateutil'
```

**Comment by januarharianto** [comment]
Thanks. This is a typo in the README, should be

```r
remotes::install_github("sydney-informatics-hub/dataharvester")
```
... as it is the more up-to-date version after I forked it, whoops. Will fix the README

**Comment by januarharianto** [comment]
First suggested in 1970aa8a0866811aeed2713cd314c6a80ff77d33, fixed in e12a234cc68bd05ccf167666220198f96d2c3b16.
